plugin/matchindent.vim

```diff
@@ -99,9 +99,9 @@ function! MatchIndent()
 	" Actually apply the rules now
 	if use_tabs > 0
 		set noexpandtab
-		set shiftwidth=4
-		set softtabstop=4
-		set tabstop=4
+		set shiftwidth=g:matchindent_tabsize
+		set softtabstop=g:matchindent_tabsize
+		set tabstop=g:matchindent_tabsize
```

Comment:
Can you provide a default value for this variable? If it's not defined then this will barf out some errors.